src/NzbDrone.Core/Download/Clients/QBittorrent/QBittorrentSettings.cs

```diff
@@ -63,6 +65,15 @@ public QBittorrentSettings()
         [FieldDefinition(10, Label = "Initial State", Type = FieldType.Select, SelectOptions = typeof(QBittorrentState), HelpText = "Initial state for torrents added to qBittorrent")]
         public int InitialState { get; set; }
 
+        [FieldDefinition(11, Label = "Remove stalled downloads", Type = FieldType.Checkbox, Advanced = true, HelpText = "Remove stalled downloads from the client. BEWARE: This could get you banned from trackers!")]
+        public bool RemoveStalledDownloads { get; set; }
+
+        [FieldDefinition(12, Label = "Stalled age threshold", Type = FieldType.Textbox, Advanced = true, HelpText = "Age threshold before a stalled torrent is considered for deletion (in seconds)")]
+        public long StalledThreshold { get; set; }
+
+        [FieldDefinition(13, Label = "Stalled inactivity threshold", Type = FieldType.Textbox, Advanced = true, HelpText = "Inactivity threshold before a stalled torrent is considered for deletion (in seconds)")]
```

Comment:
Do these make more sense in `hours` or `days` to avoid users having to calculate?

Reply:
I see your point, but this being a set-once-and-forget parameter, behind the advanced subset, I would rather keep it granular enough to all cases than restrict it to a too coarse of a unit. In any case, if you think hours is more appropiate, you are probably more familiar with the user's scenarios.

Reply:
split the difference, minutes?

Reply:
minutes works